src/typing_extensions.py

```diff
@@ -1277,7 +1277,10 @@ def _set_default(type_param, default):
         type_param.__default__ = tuple((typing._type_check(d, "Default must be a type")
                                         for d in default))
     elif default != _marker:
-        type_param.__default__ = typing._type_check(default, "Default must be a type")
+        if isinstance(type_param, ParamSpec) and default is ...:  # ... not valid <3.11
+            type_param.__default__ = default
+        else:
+            type_param.__default__ = typing._type_check(default, "Default must be a type")
```

Comment:
Aha, so the issue is that `typing._type_check` changed in Python 3.11 so that it would accept an ellipsis, but didn't allow it on previous versions